index.html

```diff
@@ -11,9 +11,223 @@
   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css" 
   integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" 
   crossorigin="anonymous">
+  <link rel="stylesheet" href="style.css">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" 
+  integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" 
+  crossorigin="anonymous">
   <title>Psych Conf</title>
 </head>
 <body>
-  
+  <main>
+    <header>
+      <div class="topnav" id="myTopnav">
+        <img class="img-responsive logo" 
+        src="images/psych-logo.png" alt="Psych Conference Logo" />
+        <a href="#project">Project</a>
+        <a href="about.html">About</a>
+        <a href="index.html">Home</a>
+        <a href="javascript:void(0);" class="icon menuToggle" id="menuToggle">
+          <i class="fa fa-bars fs-3"></i>
+        </a>
+      </div>
+    </header>
+    <!-- THE FIRST CONTENT, HOLDING THE TITLE, DESCRIPTION AND THE LOCATION OF THE EVENT -->
+    <div class="content">
+      <div class="program-info" id="program">
+        <h6>"Where spoken words become powerful!"</h6>
+        <h1>SPOKEN WORDS AND MENTAL HEALTH | CONFERENCE 2021</h1>
+        <P class="program-definition">More often than not, we speak without actually knowing the implications of our speech, 
+          We invite you to participate in the three-day conference, send in your key notes, 
+          have the stage, tell your experiences and get feedbacks. This is also a moment of reflection to see how 
+          this has affected you and who your speech has affected and find ways to curb this harmful behaviour.</P>
+        <div class="dateandlocation" >
+          <h3> 2021.17.10(SUN) ~ 20(WED)</h3>
+          <p class="location">@ Sheraton Hotel and Suites, Abuja</p>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] Note that the pages use the exact same colors, for text, backgrounds, borders, etc, from the original design. Therefore, kindly find a similar pattern to use as a background for this headline section. Using an image does not comply with the original design.

Reply:
Well done working on this part 💯